### PR TITLE
param filepath in reader and writer

### PR DESCRIPTION
**Pull request by Zluurk**

### CODE REVIEW

File: pypeman/nodes.py [hunk 1]
```diff
@@ -299,6 +321,7 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
     def process(self, msg):
+        self.channel.logger.log(self.lvl, '%s %s', str(self), repr(msg))
```
Comment (jrmi) [inline pypeman/nodes.py]: Duplicate line

File: pypeman/tests/test_debug.py [hunk 2]
```diff
@@ -44,12 +44,11 @@ def setUp(self):
         print("added handler")
 
     def tearDown(self):
-        setup_settings(SETTINGS_MODULE)
+        setup_settings(SETTINGS_MODULE)
```
Comment (jrmi) [inline pypeman/tests/test_debug.py]: Here

File: pypeman/tests/test_debug.py [hunk 3]
```diff
@@ -44,12 +44,11 @@ def setUp(self):
...
 
     def tearDown(self):
-        setup_settings(SETTINGS_MODULE)
+        setup_settings(SETTINGS_MODULE) 
         teardown_settings()
 
     def test_log(self):
-        """ can modify log config
-
+        """ can modify log config
```
Comment (jrmi) [inline pypeman/tests/test_debug.py]: And here

Comment (jrmi) [inline pypeman/tests/test_debug.py]: Dont touch it

File: pypeman/channels.py [hunk 4]
```diff
@@ -525,25 +525,29 @@ class FileWatcherChannel(BaseChannel):
...
+        if os.path.exists(self.basedir):
+            for filename in os.listdir(self.basedir):
                 if self.re.match(filename):
-                    filepath = os.path.join(self.path, filename)
+                    filepath = os.path.join(self.basedir, filename)
                     mtime = os.stat(filepath).st_mtime
                     self.data[filename] = mtime
         else:
-            self.logger.warning('path not exist: %r', self.path)
+            self.logger.warning('path not exist: %r', self.basedir)
```
Comment (jrmi) [inline pypeman/channels.py]: `Path doesn't exists: %r`

Comment (jrmi) [inline pypeman/channels.py]: Not done ;-)

Comment (jrmi) [inline pypeman/channels.py]: And uppercase

Comment (ElBui) [inline pypeman/channels.py]: Path does not exisT =)

**Pull request merged**